apps/condo/domains/acquiring/schema/GeneratePaymentLinkService.js

```diff
@@ -314,7 +314,7 @@ const GeneratePaymentLinkService = new GQLCustomSchema('GeneratePaymentLinkServi
                         throw new GQLError(ERRORS.UNPUBLISHED_INVOICE, context)
                     }
 
-                    paymentLinkBaseUrl.searchParams.set(invoicesQp, map(invoicesModels, 'id').join(','))
+                    paymentLinkBaseUrl.searchParams.set(invoicesQp, map(invoicesModels, 'id').sort().join(','))
```

Comment:
why do you need to sort ids?
Was extra `NlogN` worth it?)

Reply:
To prevent tests from random failing in the case of 2+ invoices.

Reply:
refactored